external/rdkit_cstdint.patch

```diff
@@ -0,0 +1,24 @@
+diff --git a/Code/ForceField/MMFF/StretchBend.h b/Code/ForceField/MMFF/StretchBend.h
+index 4eceb3e7e..765fabde6 100644
+--- a/Code/ForceField/MMFF/StretchBend.h
++++ b/Code/ForceField/MMFF/StretchBend.h
+@@ -10,6 +10,7 @@
+ #ifndef __RD_MMFFSTRETCHBEND_H__
+ #define __RD_MMFFSTRETCHBEND_H__
+ 
++#include <cstdint>
```

Comment:
Should we push these changes upstream (and keep using this patch for now)?  I got similar errors when I was testing local builds of RDKit.  I assumed it was something weird about my setup, but it seems like it's a real problem if the GitHub builder is having issues.

Reply:
Yeah, will do.